Model/Resolver/OrderTotal.php

```diff
@@ -0,0 +1,209 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace ScandiPWA\SalesGraphQl\Model\Resolver;
+
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Query\ResolverInterface;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+use Magento\Sales\Api\Data\OrderInterface;
+use Magento\SalesGraphQl\Model\Resolver\OrderTotal as SourceOrderTotal;
+
+
+/**
+ * Resolve order totals taxes and discounts for order
```

Comment:
Standard docs (with package class etc)
Also, please add a comment on which methods got changed in the override